lib/python/qmk/constants.py

```diff
@@ -33,8 +33,6 @@
     # 'MK20DX128'
```

Comment:
The `cortex-*` MCUs should be removed from this list.

Reply:
b4c154b95218b7b9b496231fad0945b56b2ad5fa ?